oktapam/constants/descriptions/attributes.go

```diff
@@ -23,6 +23,14 @@ var (
 	ADTaskIsActive                   = "If `true`, enables AD task"
 	ADTaskRunTest                    = "If `true`, test is performed based on specified AD Task Settings"
 	ADTaskStartHourUTC               = "If AD task is scheduled to run daily, then specify start hour in UTC"
+	ADUserSyncTaskSettingsBaseDN          = "Specifies where the rule searches for users."
+	ADUserSyncTaskSettingsFrequency       = "Frequency of the AD User Sync Task"
+	ADUserSyncTaskSettingsIsActive        = "If `true`, enables AD user sync task"
+	ADUserSyncTaskSettingsLDAPQueryFilter = "Specifies the specific criteria used to filter users."
+	ADUserSyncTaskSettingsRunTest         = "If `true`, test is performed based on specified AD User Sync Task Settings"
+	ADUserSyncTaskSettingsSIDField        = "Name of the AD SID field in User Sync queries"
```

Comment:
May be change this to => AD Attribute mapped to user security identifier that can be used by ASA to include in the passwordless certificate.

Reply:
We can revisit these later once official documentation is posted.

Reply:
Yes, these probably should be in sync with docs.